### DISCUSSION TITLE
v4.2.0  httprunner 创建新项目时报错，怎么解决？

### DISCUSSION THREAD

**Discussion by fugd1** [httprunner/httprunner#1473] [2 - 答疑（Q&A）]
(venv) PS D:\MyData\fugd1\PycharmProjects\demo> httprunner startproject my_demo
usage: httprunner [-h] [-V] {run,make} ...
httprunner: error: argument {run,make}: invalid choice: 'startproject' (choose from 'run', 'make')

**Comment by fugd1** [comment]
虚拟环境下创建新项目怎么一直报这个错误，有大佬遇到过吗
usage: httprunner [-h] [-V] {run,make} ...
httprunner: error: argument {run,make}: invalid choice: 'startproject' (choose from 'run', 'make')

**Reply by k-ming** [reply]
如果是在windows系统下，你需要先下载 httprunner.msi 程序进行安装，配置环境变量，才能使用 httprunner startproject my_demo  命令创建项目, 别的博主安装教程可以参考下
https://blog.csdn.net/weixin_45451320/article/details/125527020
